test/fixtures/cookbooks/test/recipes/single_node.rb

```diff
@@ -1,4 +1,9 @@
 apt_update 'update' if platform_family?('debian')
+
+node.default['graphite']['version'] = '1.1.6'
```

Comment:
This should just become the default if we're shipping a major version. Get folks to current